### DISCUSSION TITLE
Vercel /tmp directory

### DISCUSSION THREAD

**Discussion by guykogel** [vercel/community#6312] [Help] [answered]
### Question

We would like to keep our currencies object under the tmp directory and update it on a daily basis.
My question: is it safe to keep such a sensitive file under the tmp directory? (we need it to be under tmp in order to rewrite it every day)

**Comment by amyegan** [comment, marked as answer]
Hey @guykogel. The `/tmp` directory is ephemeral so you should use a stable data store outside of the Serverless Function if you need to reuse files between function invocations.

In situations where you only need to change a couple of variables or flags, [Edge Config](https://vercel.com/docs/storage/edge-config) might be a quick solution. There are also many integrations available if you need a full database solution: https://vercel.com/integrations#databases
